packages/api/cms-api/src/user-permissions/decorators/affected-content-scope.decorator.ts

```diff
@@ -0,0 +1,9 @@
+import { SetMetadata } from "@nestjs/common";
+
+export type AffectedContentScope = {
+    argsSelector: string;
```

Comment:
Would this work if the scope were nested in an input? For instance "input.scope"?

Reply:
Yes, it does: https://lodash.com/docs/4.17.15#get